pkg/disttask/framework/scheduler/interface.go

```diff
@@ -105,8 +104,10 @@ type Extension interface {
 
 	// GetEligibleInstances is used to get the eligible instances for the task.
 	// on certain condition we may want to use some instances to do the task, such as instances with more disk.
-	// The bool return value indicates whether filter instances by role.
-	GetEligibleInstances(ctx context.Context, task *proto.Task) ([]*infosync.ServerInfo, bool, error)
+	// if returned instances is empty, it means all instances are eligible.
+	// TODO: run import from server disk using framework makes this logic complicated,
+	// the instance might not be managed by framework.
+	GetEligibleInstances(ctx context.Context, task *proto.Task) ([]string, error)
```

Comment:
Maybe we can remove this interface?

Reply:
not now, server disk import requires this